src/librustdoc/html/static/settings.css

```diff
@@ -4,7 +4,6 @@
 }
 
 .setting-line > div {
-	max-width: calc(100% - 74px);
```

Comment:
Why is this removed?

Reply:
With the `<select>` element being aligned to the right, this rule made it awkwardly stand 74 pixels away from the end of the page.

Before:
![image](https://user-images.githubusercontent.com/11479594/95682879-2cfc9100-0be8-11eb-8c88-52300260027d.png)

After:
![image](https://user-images.githubusercontent.com/11479594/95682890-3a198000-0be8-11eb-8815-76465ff4fdc2.png)


It didn't really make sense to me so I removed it. Note that it wasn't really useful before because all the content was aligned to the left anyway, so I don't think it broke anything else.